tests/Unit/Core/ConstraintValidator/TypedRegexValidatorTest.php

```diff
@@ -270,55 +270,21 @@ public function testItFailsForIsbnTypeWhenInvalidCharacterGiven(string $invalidC
         $this->assertViolationIsRaised(new TypedRegex('isbn'), $invalidChar);
     }
 
-    /**
```

Comment:
I believe you have removed a lot of unit tests.

Reply:
Still don't know what happened. Should be good now! 😅